### PR TITLE
Made the projects list scroll when there are lots of items.

### LINKED ISSUE #903: Projects page doesn't scroll vertically
As the list of projects grows, we push the `Create new project` button down and down, and eventually it just drops off the page.  We should have overflow scroll so we don't lose it, and maybe consider moving it to the top?

screenshot 2015-08-20 18 28 01

cc @flukeout

### PR DESCRIPTION
**Pull request by flukeout**
Fixes #903

### CODE REVIEW

File: public/stylesheets/projects-style.css [hunk 1]
```diff
@@ -4,6 +4,15 @@ body {
   overflow: hidden;
 }
 
+.project-list-wrapper {
+  position: fixed;
+  overflow: scroll;
```
Comment (humphd) [inline public/stylesheets/projects-style.css]: I would use `auto` so you get scrollbars showing at least.

**Comment by humphd** [comment]
One suggestion, r=me otherwise.

File: public/stylesheets/projects-style.css (hunk 1 above)
Comment (flukeout) [inline public/stylesheets/projects-style.css]: You don't see scrollbars? I do. They only show up if you have a mouse plugged in though, like other pages.

Comment (humphd) [inline public/stylesheets/projects-style.css]: No, but it does scroll.  So either way, let's get a fix in.

**Pull request merged**